test/remote/gateways/remote_adyen_test.rb

```diff
@@ -663,6 +710,16 @@ def test_failed_refund
 
   def test_successful_credit
     response = @gateway.credit(@amount, @credit_card, @options)
+
+    assert_success response
+    assert_equal 'Received', response.message
+  end
+
+  def test_successful_credit_with_bank_account
```

Comment:
If we want to test that the correct field name is used, you can utilize `stub_comms` and `.check_request`,  ala the `test_installments_sent` test. This will help catch any regressions for that special logic.